frontend/src/views/ElectedCanteens.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="text-left">
+    <BreadcrumbsNav />
+    <h1 class="text-h6 mt-n2 mb-2 font-weight-bold">
+      Les cantines de mon territoire
+    </h1>
+    <p>Établissements dans {{ departmentsString }}</p>
```

Comment:
Add a link to ManagementPage, for people with canteens? Or a link to new canteen page for people who don't manage canteens but want to add one? Like the link you've added on the ManagementPage. Or add it to the breadcrumbs...

Reply:
+1 for the breadcrumbs

Reply:
Added here : https://github.com/betagouv/ma-cantine/pull/2904/commits/e7dcbcc220d4db40d0866e2c59deeab6615b52fd